thirdeye/thirdeye-frontend/tests/integration/pods/components/shared/common-tabs/component-test.js

```diff
@@ -0,0 +1,38 @@
+import { module, test } from 'qunit';
```

Comment:
So for testing interaction of tabs and tabpanels, I assume RCA acceptance test will handle that, correct?

Reply:
Yes. Existing has been updated.